src/constants/chains.ts

```diff
@@ -0,0 +1,11 @@
+export enum SupportedChainId {
+  MAINNET = 1,
+  RINKEBY = 4,
+  XDAI = 100,
```

Comment:
Nitpick: for legacy reasons we haven't renamed this in the other apps, but since here we are "starting from scratch", I think it's safe to name it GCHAIN or GNOSIS_CHAIN or GC. Not sure which is best.

Or would that break anything in the consumer side? 🤔

Reply:
> Or would that break anything in the consumer side? 🤔

Not that I'm aware of. Let's go with `GNOSIS_CHAIN`, this isn't exposed yet so we could rename it later